lib/trento/domain/cluster/lifespan.ex

```diff
@@ -1,14 +1,28 @@
 defmodule Trento.Domain.Cluster.Lifespan do
-  @moduledoc false
+  @moduledoc """
+  Cluster aggregate lifespan.
+
+  It controls the lifespan of the aggregate GenServer representing a cluster.
+  """
 
   @behaviour Commanded.Aggregates.AggregateLifespan
 
-  alias Trento.Domain.Events.ClusterRolledUp
+  alias Commanded.Aggregates.DefaultLifespan
+
+  alias Trento.Domain.Events.ClusterRollUpRequested
 
-  def after_event(%ClusterRolledUp{applied: false}), do: :stop
-  def after_event(_), do: :infinity
+  @doc """
+  The cluster aggregate will be stopped after a ClusterRollUpRequested event is received.
```

Comment:
`aggregate is stopped after..`

Reply:
intentionally in future form, because it is something that happens eventually. (Copied from commanded lifespan docs)

Reply:
Ok, they might have a different way of writting docs I guess.
I find the present term applicable anyway XD